crypto/modes/ocb128.c

```diff
@@ -74,7 +74,7 @@ static void ocb_double(OCB_BLOCK *in, OCB_BLOCK *out)
      */
     mask = in->c[0] & 0x80;
     mask >>= 7;
-    mask *= 135;
+    mask = -mask & 0x87;
```

Comment:
Suggested `(0 - mask)` was not a coincidence. Microsoft compiler complains about unary minus with unsigned type. It's a warning, but we attempt to minimize warnings.

Reply:
Done again.